tyr/policies/stackdriver_conditions.json

```diff
@@ -0,0 +1,198 @@
+{ 
+    "_disk_gt_70pct_10m": {
```

Comment:
Whoa, does this mean we could just add new conditions here and the appropriate alerting policies would be automatically created?